multiqc_sarek_project.sh

```diff
@@ -44,9 +44,15 @@ source activate NGI
 python "$SCRIPTS_DIR/sample_list_for_multiqc.py" --path "${PROJECT_PATH}"
 check_errors $? "Something went wrong when making the sample list"
 
+# Generate extra stats and perform qc check
+python "$SCRIPTS_DIR/multiqc_extra_stats_qc.py" --analysis_dir "${PROJECT_PATH}" --project "${PROJECT_ID}"
+check_errors $? "Something went wrong when performing qc check"
+
 mqc_content="$PROJECT_PATH/multiqc_custom_content"
-mkdir -p "$mqc_content"
+qc_content="$PROJECT_PATH/multiqc_qc_check"
```

Comment:
Unfortunately they have to be separated from the other ones or they will end up in both reports. However, we could remove the multiqc_qc_check and multiqc_custom_content at the end of the script to clean up. Normally you don't need these folder when the report has been generated.

Reply:
unfortunately yes, otherwise all content will be visible in both reports. However, we could remove the qc_check and custom_content folder at the end of the script to clean up. These folders are not really needed once the report is generated.

Reply:
Ah I see, I think it is fine to keep it as it is for now.